mod_reforged/hooks/items/shields/shield.nut

```diff
@@ -10,13 +10,13 @@
 	{
 		local ret = __original();
 
-		if (this.getDefenseFractionFromFatigue() != 0.0)
+		if (this.RF_getDefenseMult() != 1.0)
 		{
 			ret.push({
 				id = 8,
 				type = "text",
 				icon = "ui/icons/fatigue.png",
-				text = ::Reforged.Mod.Tooltips.parseString("Except when using [Shieldwall|Skill+shieldwall], the defense values drop linearly as you build [Fatigue|Concept.Fatigue] up to " + ::MSU.Text.colorizeFraction(this.getDefenseFractionFromFatigue(), {InvertColor = true}) + " at maximum [Fatigue|Concept.Fatigue]")
+				text = ::Reforged.Mod.Tooltips.parseString("Except when using [Shieldwall|Skill+shieldwall], the defense values drop linearly as you build [Fatigue|Concept.Fatigue] up to " + ::MSU.Text.colorizeMult(this.RF_getDefenseMult()) + " at maximum [Fatigue|Concept.Fatigue]")
```

Comment:
Now this tooltip here displays exactly the opposite value while keeping the same text around it. This highlights how ambiguous it is.

Reply:
No it still shows the amount that's reduced because `colorizeMult` shows the reduction. But I agree the language needs improvement anyway so I'll do that in a separate commit after this PR is merged.